app/(dashboard)/dashboard/_components/ProtocolUploader.tsx

```diff
@@ -94,6 +74,8 @@ export default function ProtocolUploader({
       return;
```

Comment:
What do you say if we change the if code block to this : 
```
if (error || !success) {
      setDialogContent({
        title: 'Protocol import',
        description: 'Error importing protocol',
        progress: false,
        error: error ?? 'Unkown error occured',
      });
      return;
    }
    ````
    Now we can remove the one in line 67

Reply:
Really nice idea. Resolved here: [1ee6dd3](https://github.com/complexdatacollective/Fresco/pull/26/commits/1ee6dd34439710e0bffcf3332a85c76fe9e0ed6d)